test/CMakeLists.txt

```diff
@@ -67,6 +68,12 @@ set(CURRENT_TEST_BINARY_DIR ${CMAKE_BINARY_DIR}/testBin/)
 
 file(GLOB_RECURSE ECDAA_TEST_FILES "*.c")
 
+if(NOT USE_TPM)
```

Comment:
This should be prefixed with `ECDAA_` probably?  And maybe be called `ECDAA_TPM_SUPPORT`?